packages/cli/src/databases/repositories/execution.repository.ts

```diff
@@ -294,6 +294,16 @@ export class ExecutionRepository extends Repository<ExecutionEntity> {
 		);
 	}
 
+	async findAllPotentiallyRunningExecutionIds() {
+		const executions = await this.find({
+			where: {
+				status: In(['running', 'retrying', 'new', 'unknown']),
+			},
+			select: ['id', 'status'],
+		});
+		return executions;
+	}
```

Comment:
Name and usage suggest this should only return the IDs:

```ts
async findAllPotentiallyRunningExecutionIds() {
	return this.find({
		where: {
			status: In(['running', 'retrying', 'new', 'unknown']),
		},
		select: ['id'],
	});
}
```

Reply:
I don't see `retrying` being a status? Did you mean `waiting`?

Reply:
copilot trying to be smart...